pages/help/help_account-management.md

```diff
@@ -11,7 +11,7 @@ eleventyNavigation:
 ---
  
 - [Access the .gov registrar](#access-the-.gov-registrar)
-- [Get help with your Login.gov account](#get-help-with-your-login.gov-account)
+- [Get help with Login.gov](#get-help-with-your-login.gov-account)
```

Comment:
**comment (non-blocking)**: changing the section header text below means needing to change the target of this link

```suggestion
- [Get help with Login.gov](#get-help-with-login.gov)
```